dotCMS/src/main/java/com/dotcms/cluster/common/ClusterServerActionThread.java

```diff
@@ -56,6 +63,24 @@ public void run() {
 					}
 					connection.commit();
 				}
+				    
+				if(LicenseUtil.getLevel()==100 ){
+				    if(noLicense==0){
+				        noLicense=System.currentTimeMillis();
+				    }
+				    //no license for a minute, that is crazy!
+				    if(System.currentTimeMillis() - noLicense > 1000*Config.getLongProperty("CLUSTER_AUTO_RELICENSE_DELAY_SECONDS", 60)){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 1000 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 60 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)